users/kimurayoshimi/3.js

```diff
@@ -0,0 +1,3 @@
+const a = (b) => [...b].findIndex((value) => value === "e");
```

Comment:
ok, but no need to convert string, use string method only.

Reply:
Changed mezzot to `indexOf`. Can you confirm this for me? 
1f0d828